tests/viscoplastic_melt_fraction.prm

```diff
@@ -0,0 +1,121 @@
+# tests the melt fraction calculation in the visco plastic material
```

Comment:
```suggestion
# This test checks whether the katz2003 mantle melting model in the 
# reactive transport model produces the correct melt fraction output.
```

Reply:
@Grant-Block - Are the pressure, temperature, and other input conditions the same as other tests for the melt material model? If yes, it would be nice to note here that the calculated melt fractions should match those other tests results.

Reply:
This only tests the melt fraction output, but no reactions occur. Did you want reactions to occur as well? Because in that case, you'll have to switch on the operator splitting scheme to actually make the reactions happen (and if you do not want reactions, then ignore my comment about the composition statistics below).

Reply:
But in any case, it would be good to have one test case that makes sure the reactions work as intended.